aws/vpc/main.tf

```diff
@@ -89,7 +89,28 @@ resource "aws_nat_gateway" "nat_gateway" {
 }
 
 /*
- * Routes for private subnets to use NAT gateway
+ * Set to true create transit gateway attachments in the private subnets and
+ * route traffic to the TGW instead of NAT GW.
+ * Should be used when create_nat_gateway=false.
+ */
+#Create Transit Gateway VPC attachment for orlando-dr to attach to hub.
```

Comment:
Since this is a reusable (and public) terraform module, let's leave out any instance-specific details, even in comments:
```suggestion
```